apps/app_rpt.c

```
@@ -2971,7 +2972,9 @@ static inline void log_keyed(struct rpt *myrpt)
 		time(&myt);
 		strftime(mydate, sizeof(mydate) - 1, "%Y%m%d%H%M%S", localtime(&myt));
 		sprintf(myfname, "%s/%s/%s", myrpt->p.archivedir, myrpt->name, mydate);
-		myrpt->monstream = ast_writefile(myfname, "wav49", "app_rpt Air Archive", O_CREAT | O_APPEND, 0, 0644);
+		myformat = myrpt->p.archiveformat ? myrpt->p.archiveformat : "WAV";
```

I thought you were preserving wav49 as the default? (Personally, I wouldn't mind wav either, but just want to confirm if that's what you intended)

Also a handy macro you could use if you wanted: `myformat = S_OR(myrpt->p.archiveformat, "WAV");`

But you don't have to do that, those are just optional helper macros

---

Too much confusion about .wav49, .wav, .WAV.  I do want to preserve .wav49 ... but also know that the asterisk code changes .wav49 to .WAV.  I'll switch this back ... so we're sure what we're getting.

---

Yes, wav49 indicates it will be gsm wrapped in wav, but the file extension is always wav. Providing wav means something differently.
See note number 2 here: https://docs.asterisk.org/Latest_API/API_Documentation/Dialplan_Applications/MixMonitor/

---

Yup.  It was the _Asterisk will silently convert the extension to '.WAV'_ comment that had me confused (like was .wav49 the legacy name and, instead, one should now use .WAV)